packages/vue/src/components/JsonForms.vue

```diff
@@ -52,8 +52,9 @@ export default defineComponent({
   },
   props: {
     data: {
-      required: true,
+      required: false,
       type: [String, Number, Boolean, Array, Object] as PropType<any>,
+      default: undefined, // allow for cases like mixed schema and the data in not yet defined
```

Comment:
Is this default needed? If not required, `undefined` is the default anyways, right?

Reply:
yes that is correct let me adjust that and remove the comment there as well then, althought we have default: undefined for uischema as well as schema - do you want me to clean that up as well ?

Reply:
when I try to remove the default then I got some lint warning -so just cleaned up the comment since it is more noise than clarity 

Prop 'data' requires default value to be set.eslint[vue/require-default-prop](https://eslint.vuejs.org/rules/require-default-prop.html)